package.json

```
@@ -56,14 +58,11 @@
     ]
   },
   "ava": {
-    "babel": false,
-    "compileEnhancements": false,
-    "extensions": [
-      "ts"
-    ],
-    "require": [
-      "ts-node/register"
-    ]
+    "typescript": {
```

```suggestion
    "files": ["!**/d.ts"],
    "typescript": {
```

This should fix the test run I think.